config.w32

```diff
@@ -1,6 +1,8 @@
 // vim:ft=javascript
 
-ARG_ENABLE("psr", "enable psr support", "no");
+ARG_ENABLE("psr", "Enable psr support", "no");
+ARG_ENABLE("sanitizer", "Enable address sanitizer extension", "no")
```

Comment:
@sergeyklay I asked in windev on IRC, and they said this issue was supposed to be fixed soon, so it may work now.

Reply:
@jbboehr Should I just remove `--enable-sanitizer` from the `config.w32 ` ?

Reply:
Assuming windev actually fixed it, we should be able to remove both of `config-profile` and `sanitizer`.

Reply:
Let me do it :sunglasses: